### PR TITLE
Add handling for no optimal paths

### PR DESCRIPTION
**Pull request by brandonabela**

### CODE REVIEW

File: Logic/Path/ball_path.py [hunk 1]
```diff
@@ -55,14 +62,18 @@ def add_graph_edges(self):
...
                             distance = self.vectors.distance_from_two_points(self.white, target_hit_position)
                             self.graph.add_edge(self.white, target_hit_position, distance)
 
                             self.graph.add_edge(target_hit_position, target_ball_position, 0)
-
                             distance = self.vectors.distance_from_two_points(target_ball_position, target_hole)
                             self.graph.add_edge(target_ball_position, target_hole, distance)
+                        else:
+                            # Adding a constant distance to make such paths less favourtable than those that reach a hole
+
```
Comment (KeithGeorgeCiantar) [inline Logic/Path/ball_path.py:75]: Maybe clarify that this hits a ball with the shortest distance when no paths are available.

**Review by KeithGeorgeCiantar — COMMENTED** [review]
The small change seems logical, only one comment suggested.

File: Logic/Path/ball_path.py (hunk 1 above)
Comment (brandonabela) [reply Logic/Path/ball_path.py:75]: Done

**Review by KeithGeorgeCiantar — APPROVED** [review]
Everything seems good, very logical change.

**Pull request merged**